check_ci.sh

```diff
@@ -10,5 +10,6 @@ black --safe --quiet examples/ pymodbus/ test/
 isort .
 pylint --recursive=y examples pymodbus test
 flake8
+mypy pymodbus
```

Comment:
Please add a mypy.ini file, so we can exclude files and also track how far we are in respect of changing the source to be typed.

Reply:
https://github.com/pymodbus-dev/pymodbus/blob/dev/setup.cfg#L504

It already exists as a section in `setup.cfg`, no need for a separate file.